app/controllers/api/v1/api_controller.rb

```diff
@@ -4,6 +4,30 @@ class ApiController < ActionController::API
       include ::ClimateWatchEngine::Cors
       include ::ClimateWatchEngine::ExceptionResponses
       include ::ClimateWatchEngine::Caching
+
+      rescue_from ActiveRecord::RecordNotFound, with: :resource_not_found
```

Comment:
these methods are defined in the climate-watch-engine, with the includes in place they can be removed: https://github.com/ClimateWatch-Vizzuality/climate-watch-gems/tree/master/climate_watch_engine/app/controllers/concerns